### ISSUE TITLE
Issue downloading LPIC level 1

### ISSUE THREAD

**Issue by steve-offutt** [vassim/linuxacademy-dl#14]
When I try and download LPIC level 1 I get the following error:
```bash
python -m linuxacademy_dl --debug https://linuxacademy.com/cp/modules/view/id/2
INFO     Debug log file created at /home/steve/linuxacademy-dl/log/log_2017-02-20T16:12:49.026867.log
DEBUG    Python: CPython 2.7.12 (default, Nov 19 2016, 06:48:10) [GCC 5.4.0 20160609]
DEBUG    Platform: Linux-4.4.0-45-generic-x86_64-with-Ubuntu-16.04-xenial [#66-Ubuntu SMP Wed Oct 19 14:12:37 UTC 2016]
DEBUG    OS: Linux - Ubuntu 16.04 xenial
INFO     Logging in ...
INFO     Analyzing ...
INFO     Logged Out!
ERROR    Unknown Exception
ERROR    unicode argument expected, got 'str'
Traceback (most recent call last):
  File "/home/steve/.local/lib/python2.7/site-packages/linuxacademy_dl/cli.py", line 219, in main
    la.analyze()
  File "/home/steve/.local/lib/python2.7/site-packages/linuxacademy_dl/linux_academy.py", line 151, in analyze
    self._course.assets()
  File "/home/steve/.local/lib/python2.7/site-packages/linuxacademy_dl/course.py", line 79, in assets
    syllabus = self._parse_syllabus()
  File "/home/steve/.local/lib/python2.7/site-packages/linuxacademy_dl/course.py", line 75, in _parse_syllabus
    sp.feed(self._fetch_syllabus())
  File "/home/steve/.local/lib/python2.7/site-packages/linuxacademy_dl/parsers.py", line 65, in feed
    super(SyllabusParser, self).feed(clean_html(data))
  File "/usr/lib/python2.7/HTMLParser.py", line 117, in feed
    self.goahead(0)
  File "/usr/lib/python2.7/HTMLParser.py", line 222, in goahead
    self.handle_data("&")
  File "/home/steve/.local/lib/python2.7/site-packages/linuxacademy_dl/parsers.py", line 112, in handle_data
    self.__text_store.write(data)
TypeError: unicode argument expected, got 'str'
```

**Comment by vassim** [comment]
Thank you for reporting this issue. I've fixed it, please use the latest version (v1.0a2)

**Comment by steve-offutt** [comment]
Awesome, thank you!
